ldms/src/ldmsd/ldmsd_failover.c

```diff
@@ -2355,6 +2356,7 @@ int failover_ping_handler(ldmsd_req_ctxt_t req)
 	data.state = f->state;
 	data.conn_state = f->conn_state;
 	data.flags = f->flags;
+	data.zero = 0;
```

Comment:
Seems like this is a change from another fix?

Reply:
The failover's ping responses are not null-terminated. With the validation patch, failover won't work. To fix this, @narategithub appends the responses with zero, and I included the fix to `ldmsd_failover.c` in this pull request.